topology/University of Wisconsin/CHTC/CHTC.yaml

```diff
@@ -502,6 +508,30 @@ Resources:
     VOOwnership:
       GLOW: 100
 
+  CHTC-ap40:
+    Active: true
+    Description: The ap40.chtc.wisc.edu login node for the OSG Open Pool
+    ID: 1468
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          Name: Aaron Moate
+          ID: OSG1000015
+      Security Contact:
+        Primary:
+          Name: Aaron Moate
+          ID: OSG1000015
+    FQDN: ap40.chtc.wisc.edu
+    Services:
```

Comment:
```suggestion
    FQDN: ap40.uw.osg-htc.org
    FQDNAliases:
      - ap2007.chtc.wisc.edu    
    Services:
```
Again, the domain name is uw.osg-htc.org; note that in DNS the host still resolves as ap2007.chtc.wisc.edu so we should list it as an FQDN alias.

Reply:
```suggestion
    FQDN: ap40.chtc.wisc.edu
    FQDNAliases:
      - ap2007.chtc.wisc.edu 
      - ospool-ap2040.chtc.wisc.edu
    Services:
```

May as well add the new A record/host